languages/go/exercises/concept/using-map/.docs/introduction.md

```diff
@@ -0,0 +1,10 @@
+`map` in go is a key-value type. In other programming language, you might know map as `dict` or associative array. The value of an uninitialized map is `nil`, and you can define map as follows
```

Comment:
The [official Go documentation](https://gobyexample.com/maps) refers to it as an associative data type, so that might be nice to use here too. We can combine that with the [description in our generic map reference document](https://github.com/exercism/v3/blob/master/reference/types/map.md):

```suggestion
A `map` in Go is an associative data type that contains a collection of key/value pairs, which other languages might refer to as as dictionary or an associative array.

The value of an uninitialized map is `nil`, and you can define a map as follows:
```